cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/NewCommand.java

```diff
@@ -313,7 +313,7 @@ public void execute() {
         }
         if (Files.exists(packagePath)) {
             errStream.println("Created new package '" + guessPkgName(packageName, template)
-                    + "' at " + packagePath + ".");
+                    + "' at " + guessPkgName(packageName, template) + ".");
```

Comment:
This will give the wrong path when the path passed to `bal new` command has a hyphen or any other character disallowed for the package name. For example, `bal new test-pkg` must give the following message:
```
Package name is derived as 'test_pkg'. Edit the Ballerina.toml to change it.

Created new package 'test_pkg' at test-pkg.
```

We can use the `argPath` in place of the `packagePath`.

Reply:
got it, Im unable to test this change as the build keeps failing with the error
```> Task :ballerina-lang:annotations:ballerinaLangLibBuild FAILED
Building langlib ...
Project Dir: C:\Users\tauro\Documents\opensource\ballerina-lang\langlib\lang.annotations\build\ballerina-src
Unknown error building : C:\Users\tauro\Documents\opensource\ballerina-lang\langlib\lang.annotations\build\ballerina-src
io.ballerina.projects.ProjectException: Provided path is already within a Ballerina package: C:\Users\tauro\Documents\opensource\ballerina-lang\langlib\lang.annotations\build\ballerina-src
        at io.ballerina.projects.internal.ProjectFiles.validateBuildProjectDirPath(ProjectFiles.java:334)
        at io.ballerina.projects.internal.PackageConfigCreator.createBuildProjectConfig(PackageConfigCreator.java:56)
        at io.ballerina.projects.directory.BuildProject.load(BuildProject.java:116)
        at org.ballerinalang.stdlib.utils.BuildLangLib.main(BuildLangLib.java:91)
Exception in thread "main" io.ballerina.projects.ProjectException: Provided path is already within a Ballerina package: C:\Users\tauro\Documents\opensource\ballerina-lang\langlib\lang.annotations\build\ballerina-src
        at io.ballerina.projects.internal.ProjectFiles.validateBuildProjectDirPath(ProjectFiles.java:334)
        at io.ballerina.projects.internal.PackageConfigCreator.createBuildProjectConfig(PackageConfigCreator.java:56)
        at io.ballerina.projects.directory.BuildProject.load(BuildProject.java:116)
        at org.ballerinalang.stdlib.utils.BuildLangLib.main(BuildLangLib.java:91)

FAILURE: Build failed with an exception.
```

is there any way to resolve this, or alternate way to test it

Im on windows btw

Reply:
I tested the change locally and it ran successfully. Can you run the command `./gradlew clean :ballerina-cli:build`? If it it still failing, please push the changes so that we can run the PR checks and validate the fix further.